### PR TITLE
Increase the reconnect period when connect fails

### PR DESCRIPTION
**Pull request by AeroNotix**
/cc @puzza007

### CODE REVIEW

File: src/folsomite_graphite_client.erl [hunk 1]
```diff
@@ -87,3 +88,6 @@ maybe_close_socket(undefined) ->
     ok;
 maybe_close_socket(Sock) when is_port(Sock) ->
     ok = gen_tcp:close(Sock).
+
+increase_connection_timeout(Current) ->
+    max(Current*2, ?MAX_RECONNECT_TIME).
```
Comment (puzza007) [inline src/folsomite_graphite_client.erl]: min?

Comment (AeroNotix) [inline src/folsomite_graphite_client.erl]: derp

Comment (AeroNotix) [inline src/folsomite_graphite_client.erl]: done

**Comment by puzza007** [comment]
:+1:

**Pull request merged**